internal/controllers/machinehealthcheck/machinehealthcheck_targets.go

```diff
@@ -312,13 +303,13 @@ func (r *Reconciler) healthCheckTargets(targets []healthCheckTarget, logger logr
 		}
 
 		if nextCheck > 0 {
-			logger.V(3).Info("Target is likely to go unhealthy", "timeUntilUnhealthy", nextCheck.Truncate(time.Second).String())
+			logger.V(3).Info("Machine is likely to go unhealthy", "timeUntilUnhealthy", nextCheck.Truncate(time.Second).String())
```

Comment:
Just fyi, this is not fixed on main. Just in case you want to fix it there as well